src/utils.jl

```diff
@@ -12,3 +12,256 @@ if VERSION <= v"1.6"
 else
     import Base: allocatedinline
 end
+
+#Excerpt from https://github.com/JuliaGPU/GPUCompiler.jl/blob/v0.19.4/src/jlgen.jl
+
+using Core: MethodInstance
+# Julia compiler integration
+
+
+## world age lookups
```

Comment:
This was removed from GPUCompiler since it can give incorrect answers. Can you add a warning that this is fundamentally unsound.

Reply:
As a comment or when invoked?